assets/r/model_build_word_cloud.R

```diff
@@ -30,9 +31,14 @@ set.seed(123)
 # TODO STEM=T|F
 # if STEM: text <- tm_map(text, stemDocument)
 
-png("WORDCLOUDPATH", width = 800, height = 600, units = "px")
+png("WORDCLOUDPATH/tmp.png", width = 800, height = 600, units = "px")
 
 # Generate word cloud
 wordcloud(words = d$word, freq = d$freq, scale=c(5,0.5), min.freq = MINFREQ, max.word = MAXWORD, random.order = RANDOMORDER, colors=brewer.pal(8, "Dark2"))
 
 dev.off()
+
+# Trim the white space using magick
+image <- image_read("WORDCLOUDPATH/tmp.png")
+trimmed_image <- image_trim(image)
+image_write(trimmed_image, path = "WORDCLOUDPATH/wordcloud.png")
```

Comment:
The red circle with a bar indicates there is no newline at the end of the file. This is recommended. Simple fix.

Reply:
Fixed.